app/controllers/ideas_controller.rb

```diff
@@ -88,7 +91,8 @@ def idea_params
         :assigned_user_id,
         :participation_level,
         :status,
-        :review_date
+        :review_date,
+        :review_year, :review_month, :review_day
```

Comment:
It's nicer to keep these one item per line - makes it easier to work with in general, especially when it comes to viewing diffs.